### DISCUSSION TITLE
Welcome to OBJEX Link Discussions!

### DISCUSSION THREAD

**Discussion by salvatoreraccardi** [salvatoreraccardi/OBJEX_LINK#6] [Announcements]
## 👋 Welcome to everyone!
Let's use Discussions as a place to talk about OBJEX Link and its future. This project absolutely needs your advice and critics to make it better 💪.

📌To get started, comment below with your experience with OBJEX Link if you have bought a kit, or if you haven't, leave your opinion or question about the project.

**Comment by jansolarski** [comment]
Hi Salvatore,
thank you very much for sharing your great work on github!

You use the connector BM28B0.6-16DS / 2-0.35V (51). Its mechanics are specified for 10TIMES INSERTIONS AND EXTRACTIONS.
For some makers, that is not enough. There are connectors for 30TIMES INSERTIONS AND EXTRACTIONS. E.g. DF40C- * DS-0.4V (51).
That would make the whole board even better.

 Thanks again!

**Reply by salvatoreraccardi** [reply]
Hi jansolarski,

I have discussed this in the past with other users, I have done many tests and the connectors seem to withstand even more than 10 INSERTIONS AND EXTRACTIONS consecutive. OBJEX Link v1.0 developed almost a year ago still works after several uses. This thing for me remains a mystery because it goes against what is written in the documentation of the connectors.

I thank you for the advice and surely it will be useful to improve the project. Have you used these connectors in other projects? Do you have any negative experiences with them?

Thanks again for your feedback👍

**Reply by salvatoreraccardi** [reply]
@jansolarski I can solve this problem with a "fake socket" ->  a module with female connectors on one side and male connectors on the other.🤔would be an affordable(2/3$) solution to prevent the problem.

**Reply by jansolarski** [reply]
Hi Salvatore,
 thank you for your prompt reply.

 Electronics is my hobby. Unfortunately, I myself have no significant experience with the connectors.
 I sell boards on Ebay. I think your boards are selling very well.
 
 The specification 10TIMES only means that the connector may break after 10Times. This can severely hamper commercial success.
 The indication 30TIMES INSERTIONS AND EXTRACTIONS means, that this connector endures, statistically 3TIMES more INSERTIONS AND EXTRACTIONS. This inspires more confidence in the buyer.

 The solution with "fake socket" I see commercial as counterproductive.
 
 I have already sold 20 pieces of DF40C-20DS-0.4V(51) in my Ebay-Shop for about 20 euros (including shipping).
 
 Yours sincerely, Jan Solarski

**Reply by salvatoreraccardi** [reply]
Hi Jan,
The problem with DF40C- * DS-0.4V (51) is that they are expensive (x10 compared to the connectors I use now). I understand your point of view, but in my opinion, the "fake socket" solution could make a difference. I will give this issue a lot of thought, thanks again for this great feedback.

Best 
Salvatore
